arch/sim/src/sim/sim_uart.c

```diff
@@ -363,18 +363,19 @@ static int tty_ioctl(struct file *filep, int cmd, unsigned long arg)
   struct tty_priv_s *priv = dev->priv;
   struct termios *termiosp = (struct termios *)(uintptr_t)arg;
 
-  if (!termiosp)
-    {
-      return -EINVAL;
-    }
-
```

Comment:
Should be better to include a DEBUGASSERT to avoid case where termiosp is NULL ?

Reply:
Done!